Sources/Jetworking/Client/Client.swift

```diff
@@ -197,15 +221,19 @@ public final class Client {
     }
 
     @discardableResult
-    public func delete<ResponseType>(endpoint: Endpoint<ResponseType>, parameter: [String: Any] = [:], _ completion: @escaping RequestCompletion<ResponseType>) -> CancellableRequest? {
+    public func delete<ResponseType: Decodable>(endpoint: Endpoint<ResponseType>, parameter: [String: Any] = [:], _ completion: @escaping RequestCompletion<ResponseType>) -> CancellableRequest? {
         do {
             let request: URLRequest = try createRequest(forHttpMethod: .DELETE, and: endpoint)
             return requestExecutor.send(request: request) { [weak self] data, urlResponse, error in
-                self?.handleResponse(
+                guard let self = self else { return }
+
+                ResponseHandler.evaluate(
                     data: data,
                     urlResponse: urlResponse,
                     error: error,
                     endpoint: endpoint,
+                    configuration: self.configuration,
```

Comment:
Do we need more than the queue? If not maybe we could pass just the queue.

Reply:
There is also a `Decoder` wrapped in `Configuration`

Reply:
You're right!